starter-code/scripts/models/article.js

```diff
@@ -40,23 +40,33 @@ Article.loadAll = function(inputData) {
 };
 
 /* This function below will retrieve the data from either a local or remote
- source, process it, then hand off control to the View: */
+source, process it, then hand off control to the View:   $.getJSON()     */
 Article.fetchAll = function() {
   if (localStorage.blogArticles) {
     /* When our data is already in localStorage:
     1. We can process and load it,
     2. Then we can render the index page.  */
+    var parsedData = JSON.parse(localStorage.blogArticles);
+    Article.loadAll(parsedData);
+    articleView.renderIndexPage();
```

Comment:
Good! We parse the string into an object, and pass it into loadAll, then we can render the page with out articles. 😎